sql/telemetry_derived/clients_histogram_aggregates_v1/query.sql

```diff
@@ -31,7 +68,8 @@ CREATE TEMP FUNCTION udf_merged_user_data(old_aggs ANY TYPE, new_aggs ANY TYPE)
         key,
         process,
         agg_type,
-        udf.map_sum(ARRAY_CONCAT_AGG(aggregates)) AS histogram_aggregates
+        ARRAY_CONCAT_AGG(old_aggregates) AS old_aggregates,
```

Comment:
At this point, `old_aggregates` would be the copy of `new_aggregates` from `old_aggs` and this was already map_summed the previous day. `old_aggregates` from `new_aggs` would be `NULL`. 

So actually, `ARRAY_CONCAT_AGG()` doesn't really do anything here but I just added it since an aggregation function is needed when not grouping by `old_aggregates`